packages/app/features/send/confirm/screen.tsx

```
@@ -207,16 +190,21 @@ export function SendConfirm() {
       log('gasEstimate', usdcFees)
       log('feesPerGas', feesPerGas)
       log('userOp', _userOp)
-      const receipt = await sendUserOp({
-        userOp: _userOp,
+      const chainId = baseMainnetClient.chain.id
+      const entryPoint = entryPointAddress[chainId]
+
+      const signature = await signUserOp({
+        userOp,
+        chainId,
         webauthnCreds,
+        entryPoint,
       })
-      assert(receipt.success, 'Failed to send user op')
-      setSentTxHash(receipt.receipt.transactionHash)
-      if (selectedCoin?.token === 'eth') {
-        await ethQuery.refetch()
-      } else {
-        await tokensQuery.refetch()
+      userOp.signature = signature
+
+      const workflowId = await transfer({ userOp })
+
+      if (workflowId) {
```

are there cases where we don't get a workflowId? we should throw an error if we don't get one back.

---

we throw in the temporal api 
https://github.com/0xsend/sendapp/blob/d623de1d591e6954151683b69c362e0ae2a9ba86/packages/api/src/routers/temporal.ts#L27-L48

the catch should handle it. Though we could probably clean it up